Bottom: 7abf17eb4d91e19582471e43dd856ccec8678c5b
Top:    390a619f0e8aa1e599fca9d37e673bc4adcc46be
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 13:35:47 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 5add1f6..2026fc4 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -695,7 +695,7 @@ static int sun_mount(struct autofs_point *ap, const char *root,
 		rv = mount_nfs->mount_mount(ap, root, mountpoint, strlen(mountpoint),
 					    what, fstype, options, mount_nfs->context);
 	} else {
-		if (loclen)
+		if (!loclen)
 			what = NULL;
 		else {
 			what = alloca(loclen + 1);
